sellercenter-category/src/main/java/com/gfg/sellercenter/category/reader/HttpReader.java

```diff
@@ -0,0 +1,61 @@
+package com.gfg.sellercenter.category.reader;
+
+import lombok.AllArgsConstructor;
+import org.json.JSONArray;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The HttpReader will do a request via HTTP
+ * @author Dennis Munchausen <dennis.munchausen@global-fashion-group.com>
+ * @version 1.0
+ * @since 1.0
+ */
+@AllArgsConstructor
+public class HttpReader {
+
+    private final String fqdnUrl;
+
+    /**
+     * This json  object represent the actual setting value only on a global level
+     * @param productIds Description name of the field in the database (global setting)
+     * @return JSONArray It retrieves a json object with main important values for a setting
+     * @throws Exception throws an exception when the endpoint is not reachable or no setting were found
+     */
+    public JSONArray getProductCategoriesByProductIds(List<Integer> productIds) throws Exception
+    {
+        String productIdsConverted = productIds.stream().map(String::valueOf)
+                .collect(Collectors.joining(","));
+
+        return new JSONArray(loadResponse("?product_ids=[" + productIdsConverted + "]"));
+    }
+
+    private String loadResponse(String extraQuery) throws IOException
+    {
+        String url1 = "/api/categories-by-product-ids/v1/";
```

Comment:
Please, use other name pattern like maybe `path`?

Reply:
Also, please make sure that exact this URL (with trailing slash) works.

Reply:
I tested it and it works ;)

Reply:
I see. Then it inconsistency in graph router (cc @yhaskell ). All other urls there are not reachable with slash at the end and this only one does not reachable without slash. 
I would vote for direct push in graph service to remove this trailing slash and also removing this trailing slash here.

Reply:
Ah sorry i miss understood something. Without `/` at the end it doesn´t work.